src/index.html

```diff
@@ -10,8 +10,176 @@
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
+
+
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset class="part">
+        <legend>Personal information</legend>
+
+        <label class="form-field">
+          Surname:
+          <input
+            type="text"
+            name="Surname:"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="form-field">
+          Name:
+          <input
+            type="text"
+            name="Name:"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="form-field">
+          How old are You?
+          <input
+            type="number"
+            name="How old are You?"
+            value="12"
+            min="1"
+            max="100"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="form-field">
+          Full date of birth:
+          <input
+            type="date"
+            name="Full date of birth:"
+          >
+        </label>
+
+        <label class="form-field, last" >
```

Comment:
there's no need in using additional separators to apply classes to your element, only one space is needed
```suggestion
        <label class="form-field last" >
```